src/Features/Blockcore.Features.WebHost/KeepAliveActionFilter.cs

```diff
@@ -18,7 +18,7 @@ public class KeepaliveActionFilter : IAsyncActionFilter
         /// Initializes a new instance of the <see cref="KeepaliveActionFilter"/> class.
         /// </summary>
         /// <param name="apiSettings">The API settings.</param>
-        public KeepaliveActionFilter(ApiSettings apiSettings)
+        public KeepaliveActionFilter(WebHostSettings apiSettings)
```

Comment:
I wonder do we still need this keepalive class? what do you think?

Reply:
It's been removed!